ambari-server/src/main/java/org/apache/ambari/server/api/handlers/DeleteHandler.java

```diff
@@ -83,8 +83,8 @@ protected ResultMetadata convert(RequestStatusMetaData requestStatusMetaData) {
       return null;
     }
 
-    if (requestStatusMetaData.getClass() != DeleteStatusMetaData.class) {
-      throw new IllegalArgumentException("RequestStatusDetails is not of type DeleteStatusDetails");
+    if (!DeleteStatusMetaData.class.isInstance(requestStatusMetaData)) {
```

Comment:
please, use the form like `!(requestStatusMetaData instanceof DeleteStatusMetaData)`

Reply:
+1. done thanks